howler/index.d.ts

```diff
@@ -1,73 +1,69 @@
-// Type definitions for howler.js v1.1.29
+// Type definitions for howler.js v2.0.0
 // Project: https://github.com/goldfire/howler.js
-// Definitions by: Pedro Casaubon <https://github.com/xperiments/>
+// Definitions by: Pedro Casaubon <https://github.com/xperiments/>, Todd Dukart <https://github.com/tdukart/>
```

Comment:
I rebased his commit here and didn't change anything. I'll address your other comments and add myself at the same time.